### PR TITLE
Simplify package structure

### PR DESCRIPTION
**Pull request by psomhorst**
This is my proposal for simplifying the package structure.

1. All loading functions (incl. binreader) are moved into a loading module. This can later be expanded for other sources of data.
2. Sequence, EITData, ContinuousData, SparseData, DataCollection, Event and Phase now live in their own files inside the eitprocessing folder. This list will probably be expanded with things like Breath. 
3. Vendor has moved to eitdata.py.

Data loading is done by loading `eitprocessing.loading.load_eit_data()`. The returned objects come from the root `eitprocessing`, e.g. `eitprocessing.sequence.Sequence`. We could reduce the verbosity e.g. by importing Sequence into the `eitprocessing` modules, shortening it to `eitprocessing.Sequence`. 

An alternative is presented in https://github.com/EIT-ALIVE/eitprocessing/tree/module_reorganisation_objects_psomhorst, where all objects like EITData and DataCollection live inside a module called objects.

### CODE REVIEW

**Comment by DaniBodor** [comment]
Hmmn, I see what you mean. I think I like the objects one best, although there are a few things I would still change.
- I think "objects" is too generic. Also, in my mind, the folder should be named functionally rather than by the fact that they hold classes/objects. So maybe something like "datahandling" or "datastructures".
- Following that logic, in my mind it does make sense to place the loading modules underneath the data handling modules. We could then still import `load_from()` into datahandling to make it `eitprocessing.datahandling.load_from()` to reduce verbosity.

I also wonder whether we should create an "analysis" folder which will then hold filters, but also parameters, rois, etc, rather than having those all directly underneath eitprocessing.
Similarly, plotting can either be renamed "outputs" or be a subfolder in outputs, which may hold all kinds of output types other than plots.

**Comment by psomhorst** [comment]
> Maybe it would be useful to have a short brainstorm meeting about this structure rather than discuss it online?

Yes, sounds good. I'll sent an e-mail to plan something.

**Comment by psomhorst** [comment]
@DaniBodor I have made the changes we talked about today. I have not adapted any tests, since you were already working on those.

File: eitprocessing/datahandling/loading/draeger.py
Comment (DaniBodor) [inline eitprocessing/datahandling/loading/draeger.py:1]: shouldn't these be named something like `load_draeger.py`, so that it's clear from the module name what it's doing? Or is it enough that it's in the loading folder?

File: eitprocessing/datahandling/loading/reader.py
Comment (DaniBodor) [inline eitprocessing/datahandling/loading/reader.py:1]: maybe call it `binreader.py` and the class `BinReader`?

File: eitprocessing/datahandling/loading/draeger.py
Comment (DaniBodor) [inline eitprocessing/datahandling/loading/draeger.py:1]: the mmap thing from #162 got lost somewhere, probably in #160 . Do you want to fix that here as well or should we make a separate PR for that?

**Review by DaniBodor — APPROVED** [review]
A few minor suggestions. Resolve as you think is best and then you can go ahead and merge.

File: eitprocessing/datahandling/event.py
[the remark this answers is not in the document]
Comment (psomhorst) [reply eitprocessing/datahandling/event.py:1]: Indeed I think it makes most sense to have Events be replaced by SparseData with strings as values in #168.

File: eitprocessing/datahandling/phases.py
[the remark this answers is not in the document]
Comment (psomhorst) [reply eitprocessing/datahandling/phases.py:1]: Same for Phases. MinData, MaxData and QRSComplex can be replaced with SparseData without values.

File: eitprocessing/datahandling/loading/draeger.py
Comment (psomhorst) [reply eitprocessing/datahandling/loading/draeger.py:1]: I think it's enough that they are in the loading folder, I think. Statement below is clear enough.

```python
from eitprocessing.datahandling.loading.draeger import load_draeger_data
```

File: eitprocessing/datahandling/loading/reader.py
Comment (psomhorst) [reply eitprocessing/datahandling/loading/reader.py:1]: Done in 2f6454783cc63a8e15e7a4bf195cb530951abb2c.

File: eitprocessing/datahandling/loading/draeger.py
Comment (psomhorst) [reply eitprocessing/datahandling/loading/draeger.py:1]: Fixed in 51ee39e3a8cdda77611ec3243752c5b93cb30571.

File: eitprocessing/datahandling/loading/__init__.py [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-from eitprocessing.eit_data.vendor import Vendor
-from eitprocessing.sequence import Sequence
+from eitprocessing.datahandling.datacollection import DataCollection
+from eitprocessing.datahandling.eitdata import EITData, Vendor
+from eitprocessing.datahandling.loading import draeger, sentec
+from eitprocessing.datahandling.sequence import Sequence
 
 
-def load_data(
+def load_eit_data(
```
Comment (DaniBodor) [inline eitprocessing/datahandling/loading/__init__.py:10]: @psomhorst, why was this change made? I thought because we are automatically loading all attached data (continuous and sparse), we decided to call it `load_data` without specifying eit

Comment (psomhorst) [reply eitprocessing/datahandling/loading/__init__.py:10]: Ah, yes, true. But we will extend `loading` with functions loading data from different sources, e.g. from a mechanical ventilator or external pressure sensors. `load_data()` would then become ambiguous.
